CONTRIBUTING.md

```diff
@@ -3,31 +3,28 @@
 This is Horusec contributing guide. Please read the following sections to learn how to ask questions and how to work on something.
 
 ## **Table of contents**
-1. [**Getting Started**](#Getting-started)
-   1. [**Before you contribute**](#Before-you-contribute)
-   2. [**Code of Conduct**](#Code-of-Conduct)
-   3. [**Legal**](#Legal)
-2. [**Prerequisites**](#Prerequisites)
-   1. [**Developer Certificate of Origin**](#Developer-Certificate-of-Origin)
-   2. [**Code Review**](#Code-Review)
-   3. [**Pull Requests**](#Pull-Requests)    
-3. [**How to contribute?**](#How-to-contribute?)
-      1. [**Prepare your development environment**](#Prepare-your-development-environment)
-      2. [**First contribution**](#First-contribution)
-      4. [**Add new feature, bugfixing or improvement**](#Add-new-feature-bugfixing-or-improvement)
-      5. [**Pull Request's approval**](#Pull-Request's-approval)
-      6. [**After your pull request's approval**](#After-your-pull-request's-approval)
-4. [**Community**](#Community)
-
-## **Getting started**
-
-### **Before you contribute**
+### 1. [**Before you contribute**](#Before-you-contribute)
```

Comment:
The following buttons are not redirecting to the content:

[**Developer Certificate of Origin**](#Developer-Certificate-of-Origin)
[**Pull Request's approval**](#Pull-Request's-approval)
[**After your pull request's approval**](#After-your-pull-request's-approval)